bin/data.js

```diff
@@ -27,6 +27,7 @@ const commands = new Set([
   defaultCommand,
   'get',
   'push',
+  'dp',
```

Comment:
it gives you help section for normalize, since we will have validate as well. `data dp validate`